### PR TITLE
added label layout, added scripts for release

### PR DESCRIPTION
**Pull request by eneufeld**
Preparation for  #548

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,34 @@
 {
-  "name": "webcomponents-jsonforms",
-  "private": true,
-  "version": "0.0.1",
+  "name": "jsonforms",
+  "version": "2.0.0-alpha.3",
+  "description": "Generating forms for JSON schema",
```
Comment (edgarmueller) [inline package.json]: I'd argue it's rather from JSON schema ;)

File: package.json [hunk 2]
```diff
@@ -1,11 +1,34 @@
 {
-  "name": "webcomponents-jsonforms",
-  "private": true,
-  "version": "0.0.1",
+  "name": "jsonforms",
```
Comment (edgarmueller) [inline package.json]: Should we already use jsonforms as the name? I think we we'd need to update the existing docs for version 1 first, as `npm install` will fetch the latest version per default.

File: package.json (hunk 1 above)
Comment (eneufeld) [inline package.json]: right, i copied the old description.

File: package.json (hunk 2 above)
Comment (eneufeld) [inline package.json]: not if you upload it with a tag, eg next

File: package.json (hunk 1 above)
Comment (eneufeld) [inline package.json]: I think we must go through all descriptions in a separate commit and make it consisten

File: package.json (hunk 2 above)
Comment (edgarmueller) [inline package.json]: Ah, ok, didn't know that.

File: package.json (hunk 1 above)
Comment (edgarmueller) [inline package.json]: I am ok with that.
